File: aspnet-core/src/GWebsite.AbpZeroTemplate.Application.Share/EvictionAsset/Dto/EvictionAssetDto.cs
using Abp.Domain.Entities;
using System;

namespace GWebsite.AbpZeroTemplate.Application.Share.EvictionAsset.Dto
{
    public class EvictionAssetDto : Entity<int>
    {
        public DateTime EvictionDate { get; set; }
        public string AssetCode { get; set; }
        public string AssetName { get; set; }
        public string Reason { get; set; }
        public string Content { get; set; }
        public bool IsReadonly { get; set; }
    }
}

File: angular/src/app/gwebsite/eviction-asset/dto/eviction-asset.dto.ts
import { ComboboxItemDto } from '@shared/service-proxies/service-proxies';

export class EvictionAssetDto {
    index?: number;
    id: number;
    evictionDate: Date | string;
    assetCode: string;
    assetName: string;
    reason: string;
    content: string;
    isReadonly: boolean;
}

export class GetEvictionAssetOutput {
    evictionAsset: EvictionAssetDto;
    evictionAssets: ComboboxItemDto[];
}
